enable_logging_API.gs

```diff
@@ -16,9 +16,7 @@ function enableLogginApisPvt(projectNumber) {
   };
   try {
     var response = UrlFetchApp.fetch(url, options);
-    var json = JSON.parse(response.getContentText());
```

Comment:
still you haven't verified that response is not having any error. Can that be case or not

Reply:
Updated